### ISSUE TITLE
Links to deleted assets still present in this repo

### ISSUE THREAD

**Issue by gmazzamuto** [dandisets/000026#4]
Take this file for example:
`rawdata/sub-I38/ses-SPIM/micr/sub-I38_ses-SPIM_sample-BrocaAreaS02_stain-Calretinin_chunk-47_SPIM.ome.tif`

This is an old asset that I deleted recently, but it still shows up in this repo as a git-annex link:
[https://github.com/dandisets/000026/blob/b3c46528117ea311e71c868e0fa2f1b88280df3e/rawdata/sub-I38/ses-SPIM/micr/sub-I38_ses-SPIM_sample-BrocaAreaS02_stain-Calretinin_chunk-47_SPIM.ome.tif](https://github.com/dandisets/000026/blob/b3c46528117ea311e71c868e0fa2f1b88280df3e/rawdata/sub-I38/ses-SPIM/micr/sub-I38_ses-SPIM_sample-BrocaAreaS02_stain-Calretinin_chunk-47_SPIM.ome.tif)

If I do
```py
ds.get_asset_by_path('rawdata/sub-I38/ses-SPIM/micr/sub-I38_ses-SPIM_sample-BrocaAreaS02_stain-Calretinin_chunk-47_SPIM.ome.tif')
```
I get:
```py
NotFoundError: No asset at path 'rawdata/sub-I38/ses-SPIM/micr/sub-I38_ses-SPIM_sample-BrocaAreaS02_stain-Calretinin_chunk-47_SPIM.ome.tif'
```

**Comment by yarikoptic** [comment]
Thank you @gmazzamuto for reporting, filed a dedicated https://github.com/dandi/dandisets/issues/174 - we will have it fixed.  Cheers

**Comment by yarikoptic** [comment]
@jwodder addressed the issue, and now we got

```shell
(base) dandi@drogon:/mnt/backup/dandi/dandisets/000026$ git log --stat -- rawdata/sub-I38/ses-SPIM/micr/sub-I38_ses-SPIM_sample-BrocaAreaS02_stain-Calretinin_chunk-47_SPIM.ome.tif
commit b0e7ff6ca1c41a50493963d2a0d06766fc71e3c6 (HEAD -> draft, github/draft, github/HEAD)
Author: DANDI User <info@dandiarchive.org>
Date:   Wed Jun 1 18:38:34 2022 +0000

    [backups2datalad] 1300 files deleted

 rawdata/sub-I38/ses-SPIM/micr/sub-I38_ses-SPIM_sample-BrocaAreaS02_stain-Calretinin_chunk-47_SPIM.ome.tif | 1 -
 1 file changed, 1 deletion(-)

commit a40cd27bbfbdac5bbe4119638255781178330580
Author: DANDI User <info@dandiarchive.org>
Date:   Wed Mar 23 01:15:33 2022 +0000

    [backups2datalad] 13098 files added

 rawdata/sub-I38/ses-SPIM/micr/sub-I38_ses-SPIM_sample-BrocaAreaS02_stain-Calretinin_chunk-47_SPIM.ome.tif | 1 +
 1 file changed, 1 insertion(+)
```
;-) So let's consider the issue addressed by b0e7ff6ca1c41a50493963d2a0d06766fc71e3c6
